packages/odyssey-react-mui/src/Breadcrumbs.tsx

```diff
@@ -101,14 +101,19 @@ const defaultTruncationValue = 5;
 const BreadcrumbList = ({
   children,
   homeHref,
-  maxVisibleItems = defaultTruncationValue,
+  maxVisibleItems: maxVisibleItemsProp = defaultTruncationValue,
   testId,
   translate,
 }: BreadcrumbsProps) => {
   const { t } = useTranslation();
 
   const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
 
+  const maxVisibleItems = useMemo(
+    () => Math.min(Math.max(maxVisibleItemsProp, 0), children.length),
+    [maxVisibleItemsProp, children],
+  );
```

Comment:
I wonder if we should either make this a custom function or hook called `clamp`. It's a common thing you'd wanna do in JavaScript; although, this is the only place we're doing it.

The reason I ask is that we're lacking unit tests, and this is exactly the kind of thing you'd wanna test separate from the component itself.

Reply:
I considered splitting this into its own function, but I think we should wait until we have a second use case, just for the sake of avoiding premature abstraction. Wdyt?

Reply:
Naw, the reason for the abstraction is for testing. I often pull out stuff that looks like it could be reusable so we can test it.

I agree, it probably shouldn't be pulled out, but if we export it from here and unit test just the clamping, it wouldn't match the rest of the codebase.

I don't have any strong preference to force a moving of this code since it's essentially just vanilla JS.